### DISCUSSION TITLE
Airflow postgres synch issue

### DISCUSSION THREAD

**Discussion by sabarnwal** [apache/airflow#39932] [General]
### Apache Airflow version

Other Airflow 2 version (please specify below)

### If "Other Airflow 2 version" selected, which one?

2.6.2

### What happened?

We are on airflow v2.6.2 , using k8s Executor. This has recently started throwing intermittent errors like:
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "ec2.XXX.com" to address: Temporary failure in name resolution.

Can anyone please help us on this issue?

### How to reproduce

Scheduled tasks are throwing this error. Even for retries , sometimes tasks are throwing this error.

### Operating System

PRETTY_NAME="Debian GNU/Linux 11 (bullseye)" NAME="Debian GNU/Linux"

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
If you have problem with your deployment, please do not open Airlfow Issues for it, at most create a discussion so that maybe someone will help you in solving your problem. Converted it so.

You should look at resources of your deployment not Airflow. It looks like your DNS has problems with resolving names but it has nothing to do with Airflow. Usually it is some form of limited resources (memory, cpu, networking) given to whatever your DNS is used in your k8s and you need to look at documetntation of your k8s installation and fix your configuration.

**Reply by sabarnwal** [reply]
@potiuk , we looked at our core-dns usage and there is no spike or resource constraints there. Hence, opened an issue.

**Reply by potiuk** [reply]
It does not change much. That error is returned by the OS not airflow so it has nothing to do with Airflow. You must dig deeper. Maybe it's not spike but simply limits you have

**Reply by Taragolis** [reply]
> dig deeper

Yeah really need [`dig`](https://linux.die.net/man/1/dig) deeper 🤣

```console
❯ dig ec2-airflow-postgres.khatabook.com
```
